app/views/components/_datetime_fields.html.erb

```diff
@@ -46,12 +47,14 @@
 %>
 
 <%= tag.div class: root_classes, data: data_attributes do %>
-  <%= render "govuk_publishing_components/components/heading", {
-    text: "Date",
-    heading_level: heading_level || 3,
-    font_size: heading_size || "m",
-    padding: true,
-  } %>
+  <% unless date_only %>
```

Comment:
Great work! Just one comment. It feels like you should still be able to provide a heading here like you can in the date_input component https://components.publishing.service.gov.uk/component-guide/date_input



Maybe we could do something like 

```
  <% unless date_only || !date_heading_label %>
    <%= render "govuk_publishing_components/components/heading", {
      text: date_heading || "Date",
      heading_level: heading_level || 3,
      font_size: heading_size || "m",
      padding: true,
    } %>
  <% end %>

```

That way you can pass it in as an option if you choose to. Wdyt?

Reply:
Yes, that makes sense. Though it needs to be AND rather than OR for the `unless` clause.

Reply:
apologies you're correct!